src/access.lua

```diff
@@ -83,6 +83,9 @@ local function encode_jwt_token(conf, payload, key)
       b64_encode(get_kong_key("pubder", get_public_key_location(conf)))
     }
   }
+  if (conf.key_id) then
```

Comment:
@row1govender Can you make 

```
if (conf.key_id) then 
```
just 
```
if conf.key_id then
```

Trying to keep consistency with our other if checks like this.

Reply:
Done